CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 2.1.1 (WIP)
+**Features**
+
+**Bug Fixes**
+- Fixed the case sensitivity of content type for file extensions.
+
+**Optimizations**
+- File is not downloaded in file_cache when truncate flag is received.
```

Comment:
- Optimized file-cache to skip download when O_TRUNC flag is provided in open call.